ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/mlag-configuration.md

```diff
@@ -38,7 +46,7 @@ interface Management1
 
 ## MLAG
 
-### MLAG Summary
+### MLAG IPv4 Summary
```

Comment:
Instead of changing + adding these sections, you could just add two extra columns in the table

Reply:
I am fine with either option, but as only one key type can ever be used the extra table columns should always be blank.

Checking the case and creating individual tables for each just kept it cleaner without any loss of fidelity.